cloudinstall/single_install.py

```diff
@@ -178,7 +223,8 @@ def _install_upstream_deb(self):
         log.debug('Found upstream deb, installing that instead')
         filename = os.path.basename(self.config.getopt('upstream_deb'))
         utils.container_run(
-            self.container_name, 'sudo dpkg -i .cloud-install/{}'.format(
+            self.container_name,
+            'sudo dpkg -i /home/ubuntu/.cloud-install/{}'.format(
```

Comment:
Can either do this way or use_ssh=True and keep it relative. I dont really have a preference which.